librz/include/rz_analysis.h

```diff
@@ -614,19 +634,39 @@ typedef enum {
 } RzAnalysisVarAccessType;
 
 typedef struct rz_analysis_var_access_t {
-	const char *reg; // register used for access
-	st64 offset; // relative to the function's entrypoint
-	st64 stackptr; // delta added to register to get the var, e.g. [rbp - 0x10]
-	ut8 type; // RzAnalysisVarAccessType bits
+	st64 offset; ///< address relative to the function's entrypoint where the access happens
+
+	/**
+	 * Register used for access.
+	 * For example when accessing some stack variable by `[rbp - 0x10]`, this will be "rbp"
+	 */
+	const char *reg;
+
+	/**
+	 * Delta added to register when the var is accessed,
+	 * For example for `[rbp - 0x10]`, this will be -0x10.
+	 */
+	st64 addend;
```

Comment:
maybe `relative_offset` is a better name ?

Reply:
I chose this name to deliberately express that is is a value that is added to the register, in direct contrast to the stack offset that is saved in the variable itself and describes the offset from the bottom of the stack.

Reply:
i still don't like it, but makes sense.